tests/chains/tendermint/simapp/test_helpers.go

```diff
@@ -442,3 +337,13 @@ type EmptyAppOptions struct{}
 func (ao EmptyAppOptions) Get(o string) interface{} {
 	return nil
 }
+
+// FundAccount is a utility function that funds an account by minting and sending the coins to the address
+// TODO(fdymylja): instead of using the mint module account, which has the permission of minting, create a "faucet" account
+func FundAccount(app *SimApp, ctx sdk.Context, addr sdk.AccAddress, amounts sdk.Coins) error {
```

Comment:
Do we need this function? Even if so, the copied todo comment should be removed.

Reply:
OK. This entire file was intentionally copied, so we should not edit this file.